docs/google.rst

```diff
@@ -31,14 +95,52 @@ Cloud Storage
 Overview
 ========
 Google Cloud Storage is a cloud file storage system. It uses buckets in which to
-store arbitrary files referred to as blobs.
+store arbitrary files referred to as blobs. You may use this connector to upload Parsons tables as blobs, download them to files, and list available blobs.
+
+To use the GoogleCloudStorage class, you will need Google Drive credentials. If you are the administrator of your Google Cloud account, you can generate them in the `Google Cloud Console APIs and Services <https://console.cloud.google.com/apis/credentials/serviceaccountkey?_ga=2.116342342.-1334320118.1565013288>`_.
+
+==========
+Quickstart
+==========
+
+To instantiate the GoogleBigQuery class, you can pass the constructor a string containing either the name of your Google Drive credentials file or a JSON string encoding those credentials. Alternatively, you can set the environment variable ``GOOGLE_APPLICATION_CREDENTIALS`` to be either of those strings and call the constructor without that argument.
```

Comment:
Again, "Google service account credentials" instead of "Google Drive credentials".

Reply:
Good catch! Changed and pushed